Dockerfile

```diff
@@ -0,0 +1,18 @@
+# base image
+FROM node:12.2.0-alpine
```

Comment:
It could be better if we use the stable version of node `node:10-alpine`. Can you change this?